### ISSUE TITLE
cordova@8.1.0 does not work on Node.js 4

### ISSUE THREAD

**Issue by brodybits** [apache/cordova-cli#339]
```sh
C:\Users\brodybits
λ cordova --version
C:\Users\brodybits\AppData\Local\nvs\node\4.9.1\x64\node_modules\cordova\bin\cordova:24
const { events, CordovaError } = require('cordova-common');
      ^

SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:140:18)
    at node.js:1043:3

C:\Users\brodybits
```

While Node.js 4 support was dropped from Cordova 9, Node.js 4 should still be supported in patch and minor update of Cordova 8.

P.S. This issue was discovered while investigating issues reported in #337 and #338.

**Comment by brodybits** [comment]
I think AppVeyor CI and Travis CI should have caught this one. This needs to be checked by AppVeyor CI and Travis CI in the future.

**Comment by janpio** [comment]
Should the title of this issue start with `cordova@8.1.0 ...` instead of `npm@8.1.0 ...`?

**Comment by janpio** [comment]
FYI: This is causing plugin CI tests to fail. Example: https://github.com/apache/cordova-cli/issues/339

**Comment by brodybits** [comment]
This issue should be resolved in version 8.1.1, closing.
